src/components/balloon-pump-game.js

```diff
@@ -0,0 +1,97 @@
+import { Screen } from "../../node_modules/genie/src/core/screen.js";
+
+export class BalloonPumpGame extends Screen {
+    constructor() {
+        super();
+    }
+
+    preload() {
+        this.keyLookup = this.layoutFactory.keyLookups[this.game.state.current];
+        // this.selectedSeason = this.context.inState.transient.option && this.context.inState.transient.option === 1 ? "winter" : "summer";
```

Comment:
can this be removed now?

Reply:
I left it in in case we need to remember how to wire up the select screen. Happy to take it out if you like.